arviz/tests/test_plots.py

```diff
@@ -233,12 +233,11 @@ def test_plot_parallel_raises_valueerror(df_trace):  # pylint: disable=invalid-n
 
 
 @pytest.mark.parametrize("model_fit", ["pymc3_fit", "stan_fit"])
-def test_plot_parallel(models, model_fit):
+@pytest.mark.parametrize("norm_method", ["normal", "minmax", "rank"])
+def test_plot_parallel(models, model_fit, norm_method):
     obj = getattr(models, model_fit)
     assert plot_parallel(obj, var_names=["mu", "tau"])
-    assert plot_parallel(obj, var_names=["mu", "tau"], norm_method="normal")
-    assert plot_parallel(obj, var_names=["mu", "tau"], norm_method="minmax")
-    assert plot_parallel(obj, var_names=["mu", "tau"], norm_method="rank")
+    assert plot_parallel(obj, var_names=["mu", "tau"], norm_method=norm_method)

```

Comment:
This one could be parametrized too. The trick here is to only have one assert in the test, and only use `pytest.mark.parametrize` once

Reply:
Done.I'll do so in the next test as well once this one passes.